lib/oli/delivery/paywall.ex

```diff
@@ -343,6 +362,24 @@ defmodule Oli.Delivery.Paywall do
     end
   end
 
+  @doc """
+  Get the last X(quantity) payment codes for the given product.
+  """
+  def list_payments_by_count(product_slug, count) do
```

Comment:
Wdyt about adding some unit tests for this function?

Reply:
Yes, but where do you think I should test it? Should I create a new test file to test that function? I don't know if it would look good to include it in the test file of the controller.

Reply:
We are adding a new function to the `paywall` context, so maybe there's already a `paywall_test` module that is unit testing some of the functions in that context (if it's not, I agree we should create it 🙂 )

Reply:
You are right! 🙂